app/controllers/videos_controller.rb

```diff
@@ -7,10 +7,30 @@ def index
 
   def show
     @video = Video.find(params[:id])
+    @review = Review.new
   end
 
   def search
     @videos = Video.search_by_title(params[:search])
   end
 
+  def review
```

Comment:
I don't know if I would put this `review` in this controller - it should really be under a `reviews_controller` as a `create` action.